base/Dockerfile

```diff
@@ -31,42 +31,50 @@ RUN python3 -m pip install -v --no-cache-dir -U -r requirements.txt
 
 FROM alpine:3.16.1
 LABEL mainainer='b3vis'
-COPY --chmod=755 entry.sh /entry.sh
-RUN apk upgrade --no-cache \
-    && apk add --no-cache \
+ARG PYTHON_VERSION=3.10
+VOLUME /mnt/source
+VOLUME /mnt/borg-repository
+VOLUME /root/.borgmatic
+VOLUME /etc/borgmatic.d
+VOLUME /root/.config/borg
+VOLUME /root/.ssh
+VOLUME /root/.cache/borg
+HEALTHCHECK --interval=30s --timeout=10s --start-period=20s --retries=3 CMD borgmatic --version || exit 1
```

Comment:
Is this really necessary? Seems pointless to me

Reply:
Not a requirement. Can be removed if there's no value-add.